core/src/main/scala/kafka/zk/ZkData.scala

```diff
@@ -445,47 +447,131 @@ object StateChangeHandlers {
   def zkNodeChangeListenerHandler(seqNodeRoot: String) = s"change-notification-$seqNodeRoot"
 }
 
+// Todo(ac): Rename ResourceNameType to PatternType (and nameType name-Type etc).
```

Comment:
Oops - this is dropped already in the next PR. I can clean this up if people would prefer?